pyspark/bigdl/optim/optimizer.py

```diff
@@ -669,6 +669,25 @@ def set_checkpoint(self, checkpoint_trigger,
         callBigDlFunc(self.bigdl_type, "setCheckPoint", self.value,
                       checkpoint_trigger, checkpoint_path, isOverWrite)
 
+    def set_constant_clipping(self, min_value, max_value):
```

Comment:
shall we change the name to gradient clipping? We can not know what it is clipping only by the method name. And add some docs?

Reply:
if the name too long, u can use sth. like set_gradclip_const, and set_gradclip_l2norm

Reply:
Sounds good

Reply:
Updated